modules/structured_light/include/opencv2/structured_light/structured_light.hpp

```diff
@@ -78,9 +78,10 @@ class CV_EXPORTS_W StructuredLightPattern : public virtual Algorithm
    @note All the images must be at the same resolution.
    */
   CV_WRAP
-  virtual bool decode( InputArrayOfArrays patternImages, OutputArray disparityMap, InputArrayOfArrays blackImages =
-                          noArray(),
-                      InputArrayOfArrays whiteImages = noArray(), int flags = DECODE_3D_UNDERWORLD ) const = 0;
+  virtual bool decode( InputArrayOfArrays patternImages, OutputArray disparityMap,
+                       InputArrayOfArrays blackImages = noArray(),
+                       InputArrayOfArrays whiteImages = noArray(),
+                       int flags = DECODE_3D_UNDERWORLD ) const = 0;
```

Comment:
GitHub shows that you've changed permissions for this file: 100644 -> 100755 (https://en.wikipedia.org/wiki/File_system_permissions#Numeric_notation).
It means that you've added execute permission for group and all users, but it does not make sense for `.hpp` file.

You need to run (in _opencv_contrib_ folder): 
```
chmod 644 modules/structured_light/include/opencv2/structured_light/structured_light.hpp
git add modules/structured_light/include/opencv2/structured_light/structured_light.hpp
git commit
```